moshi/src/main/java/com/squareup/moshi/internal/Util.java

```diff
@@ -55,6 +56,8 @@
   /** A map from primitive types to their corresponding wrapper types. */
   private static final Map<Class<?>, Class<?>> PRIMITIVE_TO_WRAPPER_TYPE;
 
+  private static final AtomicBoolean RECORDS_AVAILABLE = new AtomicBoolean(false);
```

Comment:
Normal `boolean`? Static init blocks cannot run concurrently.

Reply:
I might be missing something but java doesn't seem to like this
![image](https://user-images.githubusercontent.com/1361086/130308490-13fe1b69-ab9a-40c8-b051-1e91316d850b.png)

Reply:
Ended up removing this